src/api/cpp/cvc5.cpp

```diff
@@ -6929,6 +6950,11 @@ std::string Solver::getInfo(const std::string& flag) const
 std::string Solver::getOption(const std::string& option) const
 {
   CVC5_API_TRY_CATCH_BEGIN;
+  std::vector<std::string> options = options::getNames();
+  CVC5_API_UNSUPPORTED_CHECK(
+      option.find("command-verbosity") != std::string::npos
```

Comment:
Why this extra handling?

Reply:
@nafur do you know why `command-verbosity` is separate and not in `options`?

Reply:
Hm, I've never changed it. It has a special syntax (`(<command-name> verbosity)`), but we can do that with a option handler. I'll put in on my todo list...